src/pages/guru/academic_guru.jsx

```
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react';
+import Sidebar_guru from "../../component/sidebar/sidebar_guru.jsx"
+import './academic_guru.css';
+import UserInfo from "../../component/user-info/user-info.jsx";
+import Calendar from "../../../component/common/calendar.jsx";
+
+const AcademicGuru = () => {
+  const [currentMonth, setCurrentMonth] = useState(new Date());
```

ESLint

[error] 'currentMonth' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)